docs/spec.md

```diff
@@ -215,21 +218,21 @@ Quantized types satisfy the following constraints:
     * `storage_min = -2^(d-1)+1`, `storage_max = 2^(d-1)-1`.
   * If `is_unsigned(storage_type)`:
     * `storage_min = 0`, `storage_max = 2^d - 1`.
-* (C5) For all `i`, `type(scales[i]) = expressed_type`.
-* (C6) For all `i`, `scales[i] > 0`.
-* (C7) For all `i`, `type(zero_points[i]) = i64`.
-* (C8) `size(scales) = size(zero_points)`.
-* (C9) If `quantization_dimension` is empty, then `size(scales) = 1`.
-* (C10) If `quantization_dimension` is not empty, then
+* (C5) `type(scale) = expressed_type`.
+* (C6) `scale > 0`.
+* (C7) `type(zero_point) = i64`.
```

Comment:
I think we should still keep `scales` and `zero_points` here. I understand that in the new grammar they don't exist as top-level non-terminals, but so do `scale` and `zero_point`.

Reply:
will be restored in the next commit.